StyleCop.Analyzers/StyleCop.Analyzers.Test/NamingRules/SA1306UnitTests.cs

```diff
@@ -19,7 +19,18 @@ public class SA1306UnitTests : CodeFixVerifier
             await this.VerifyCSharpDiagnosticAsync(testCode, EmptyDiagnosticResults, CancellationToken.None);
         }
 
-        private async Task TestThatDiagnosticIsNotReportedImpl(string modifiers)
+        [Theory]
+        [InlineData("const")]
+        [InlineData("private const")]
+        [InlineData("internal const")]
+        [InlineData("protected const")]
+        [InlineData("protected internal const")]
+        [InlineData("internal readonly")]
+        [InlineData("protected readonly")]
+        [InlineData("protected internal readonly")]
+        [InlineData("public")]
+        [InlineData("internal")]
+        public async Task TestThatDiagnosticIsNotReported(string modifiers)
```

Comment:
I would add this to the other issue I suggested creating. Mention that the tests do not cover all combinations of modifiers but should.